packages/blockchain-link/src/index.ts

```diff
@@ -1,12 +1,14 @@
 import { EventEmitter } from 'events';
+import { createDeferred } from '@trezor/utils';
```

Comment:
```suggestion
import { createDeferred, Deferred } from '@trezor/utils';
```
no need for separate `import type` (below) if you are importing createDeferred in runtime anyway

Reply:
on second thought, maybe it should be `from '@trezor/utils/lib/promise'`?
beacuse `promise` is the only @trezor/utils used accros `blockchain-link` package, som maybe it is unnecessary to bundle whole `@trezor/utils` index here?

Reply:
:+1: done